SUMMARY.md

```diff
@@ -24,6 +24,14 @@
     * [Forking Ocean Market](building-with-ocean/build-a-marketplace/forking-ocean-market.md)
     * [Customising your market](building-with-ocean/build-a-marketplace/customising-your-market.md)
     * [Deploying your market](building-with-ocean/build-a-marketplace/deploying-market.md)
+  * [Obtaining Ethereum node provider API key](building-with-ocean/obtaining-api-key.md)
```

Comment:
I think this page is too short with too little content to deserve its own page. Can we move the content into the top of the configuration page?